src/dswx_sar/masking_with_ancillary.py

```diff
@@ -190,27 +187,20 @@ def check_water_land_mixture(args):
     water_mask : numpy.ndarray
         Processed water mask.
     """
-    i, size, minimum_pixel, bounds, int_linear_str, water_label_str, \
-        water_mask_str, pol_ind = args
-
-    x_off = bounds[0]
-    y_off = bounds[2]
-    width = bounds[1] - bounds[0]
-    height = bounds[3] - bounds[2]
-    window = Window(x_off, y_off, width, height)
-
-    # read subset of water map and intensity from given bounding box
-    with rasterio.open(int_linear_str) as src:
-        int_linear = src.read(window=window)
-        int_linear = np.squeeze(int_linear[pol_ind, :,:])
-
-    with rasterio.open(water_label_str) as src:
-        water_label = src.read(window=window)
-        water_label = np.squeeze(water_label)
-
-    with rasterio.open(water_mask_str) as src:
-        water_mask = src.read(window=window)
-        water_mask = np.squeeze(water_mask)
+    i, size, minimum_pixel, bounds, int_linear_block, water_label_block, \
```

Comment:
Docstring above needs to be updated to reflect can from `str` paths to `np.ndarray`s.

Reply:
Fixed.